code/modules/antagonists/cult/runes.dm

```diff
@@ -202,11 +203,16 @@ structure_check() searches for nearby cultist structures required for the invoca
 		fail_invoke()
 		log_game("Offer rune failed - no eligible targets")
 		return
+	var/mob/living/L = pick(myriad_targets)
+	if(HAS_TRAIT(L, TRAIT_SACRIFICED))
+		fail_invoke()
+		log_game("Offer rune failed - target has already been sacrificed")
+		to_chat(invokers, "<span class='warning'>[L] has already been sacrificed!</span>")
+		return
```

Comment:
```suggestion
	//SKYRAT EDIT ADDITION BEGIN
	var/mob/living/L = pick(myriad_targets)
	if(HAS_TRAIT(L, TRAIT_SACRIFICED))
		fail_invoke()
		log_game("Offer rune failed - target has already been sacrificed")
		to_chat(invokers, "<span class='warning'>[L] has already been sacrificed!</span>")
		return
		//SKYRAT EDIT END
```

Reply:
Did as requested, did only one comment for one liners, it should be fine given what the documentation for modulirization says